L/Libxc/Libxc_GPU/build_tarballs.jl

```diff
@@ -55,25 +55,25 @@ dependencies = [
 
 # XXX: support only specifying major/minor version (JuliaPackaging/BinaryBuilder.jl#/1212)
 cuda_full_versions = Dict(
+    v"10.2" => v"10.2.89",
     v"11.0" => v"11.0.3",
-    v"12.0" => v"12.0.0"
+    v"12.0" => v"12.0.0",
 )
 
 # Build Libxc for all supported CUDA toolkits
 #
-# The library doesn't have specific CUDA requirements, so we only build for CUDA 11.0 and 12.0,
-# which (per semantic versioning) should support every CUDA 11.x and 12.x version.
+# The library doesn't have specific CUDA requirements, so we build for CUDA 10.1 till 11.0,
+# which (per semantic versioning) should support every CUDA 10.x and 11.x version.
+# CUDA 12 causes issues, so we don't build that yet.
 #
-# TODO Note: 12 not yet fully rolled out.
-for cuda_version in [v"11.0"], platform in platforms
+for cuda_version in [v"10.2", v"11.0", ], platform in platforms
     augmented_platform = Platform(arch(platform), os(platform); cuda=CUDA.platform(cuda_version))
     should_build_platform(triplet(augmented_platform)) || continue
 
     cuda_deps = [
         BuildDependency(PackageSpec(name="CUDA_full_jll",
                                     version=cuda_full_versions[cuda_version])),
-        RuntimeDependency(PackageSpec(name="CUDA_Runtime_jll",
-                                      version=v"0.2"), compat="0.2"),  # avoid pulling in CUDA 12 for now.
+        RuntimeDependency(PackageSpec(name="CUDA_Runtime_jll")),
```

Comment:
Without changing version number, this won't change the compat.  Is the compat not ok anyway?

Reply:
Well. This might be me getting it wrong, but as of now the pin to version 0.2 of the runtime prevents me from using CUDA.jl beyond 4.0.1, because from 4.1.0 onwards it needs CUDA_Runtime_jll > 0.4. I was hoping by removing this compat to prevent this issue.

Reply:
@giordano Ok, I found out what's causing my observed behaviour, which is indeed not related to this package, so this PR becomes obsolete. Thanks!